openslides/motions/config_variables.py

```diff
@@ -161,7 +161,7 @@ def get_config_variables():
 
     yield ConfigVariable(
         name='motions_comments',
-        default_value=[{'name': 'Comment', 'public': True}],
```

Comment:
No, I want to use a simple default setup. Motion comments is an extended feature (like supporters, amendments etc.) which should be activate if really needed.